frontend/packages/console-shared/src/selectors/common.ts

```diff
@@ -0,0 +1,6 @@
+import * as _ from 'lodash-es';
```

Comment:
We should consider a different name for this folder. "selectors" has a specific meaning in k8s (label selectors)